widget/playground/src/components/Collapse/Collapse.tsx

```diff
@@ -0,0 +1,36 @@
+import type { PropTypes } from './Collapse.types';
```

Comment:
Couldn't we use Radix? Or don't we have anything in our design system?

Reply:
It was supposed to be a simple one because of the deadline. however, I used Radix and transferred it to widget/UI.